### DISCUSSION TITLE
visionOS support for PromiseKit

### DISCUSSION THREAD

**Discussion by Swati-brahma** [mxcl/PromiseKit#1345] [Q&A]
I see in your podspec file visionOS support is added, but while adding dependencies using cocoa pods getting this error:
`[!] The platform of the target `Target` (visionOS 1.0) is not compatible with `PromiseKit (8.1.1)`, which does not support `visionOS`.`

Also do not see visionOS target being added in package, so wanted to know is there a plan to add visions support on this SDK in upcoming future? 
Also, is there any workaround to add PromiseKit for visionOS target meanwhile?

Any help will be appreciated. Thanks in Advance.

**Comment by RomanPodymov** [comment]
Hello @Swati-brahma 
Please use `master` branch, add this line to your Podfile:
```
pod 'PromiseKit', :git => 'https://github.com/mxcl/PromiseKit.git', :branch => 'master'
```
